### PR TITLE
Callback scopes

### PR DESCRIPTION
**Pull request by Therzok**

### CODE REVIEW

**Comment by Therzok** [comment]
https://gist.github.com/Therzok/adccb27d8405c82ca03b5f13b76b7804

File: atk/Global.custom [hunk 1]
```diff
@@ -32,4 +32,69 @@
...
+		[DllImport("libatk-1.0-0.dll", CallingConvention = CallingConvention.Cdecl)]
+		static extern uint atk_add_focus_tracker(AtkSharp.EventListenerNative focus_tracker);
+
+		public static uint AddFocusTracker(Atk.EventListener focus_tracker) {
+			AtkSharp.EventListenerWrapper focus_tracker_wrapper = new AtkSharp.EventListenerWrapper (focus_tracker);
+			GCHandle gch = GCHandle.Alloc (focus_tracker_wrapper);
+
+			uint ret = atk_add_focus_tracker(focus_tracker_wrapper.NativeDelegate);
+
+			if (ret != 0) {
```
Comment (alanmcgovern) [inline atk/Global.custom]: if ret == 0 then we should free the GCHandle?

Comment (alanmcgovern) [reply atk/Global.custom]: Same issue in all the places :)

Comment (Therzok) [reply atk/Global.custom]: 💯

Comment (Therzok) [reply atk/Global.custom]: Fixed.

**Comment by alanmcgovern** [comment]
💯

**Pull request merged**